lib/recording/RecordingEntrySession.js

```diff
@@ -47,7 +49,8 @@ class RecordingEntrySession {
         let now = new Date();
         let entryId = entryObj.entryId;
         let recordedEntryId = entryObj.recordedEntryId;
-        if (entryObj.recordStatus === kalturaTypes.KalturaRecordStatus.APPENDED){
+        this.entryServerNodeId = entryServerNodeId;
```

Comment:
i think there were situations where entryServerNodeId has changed during the stream, also entryObj already contains entryServerNodeId... so it's duplication